# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1319023304 -3600
#      Wed Oct 19 12:21:44 2011 +0100
# Node ID 30c4bd2592dd8bb8a7c1037dd773c0a77a8d3813
# Parent  ecff4c684b894279d1e5d1d081bcd9f1d1d78388
Yet another required WIN32_LEAN_AND_MEAN.

* gl2ps.h: Define WIN32_LEAN_AND_MEAN.

diff --git a/src/gl2ps.h b/src/gl2ps.h
--- a/src/gl2ps.h
+++ b/src/gl2ps.h
@@ -41,16 +41,17 @@
 
 /* Define GL2PSDLL at compile time to build a Windows DLL */
 
 #if defined(WIN32) || defined(_WIN32) || defined(__WIN32__) || defined(__NT__)
 #  if defined(_MSC_VER)
 #    pragma warning(disable:4115)
 #    pragma warning(disable:4996)
 #  endif
+#  define WIN32_LEAN_AND_MEAN
 #  include <windows.h>
 #  if defined(GL2PSDLL)
 #    if defined(GL2PSDLL_EXPORTS)
 #      define GL2PSDLL_API __declspec(dllexport)
 #    else
 #      define GL2PSDLL_API __declspec(dllimport)
 #    endif
 #  else
